src/components/DeploymentCombobox.vue

```diff
@@ -7,15 +7,20 @@
       <slot v-bind="scope">
         <UseDeploymentSlot v-if="isString(scope.value)" :deployment-id="scope.value">
           <template #default="{ deployment }">
-            {{ deployment.name }}
+            <template v-if="deployment.flowId">
```

Comment:
Is flowId not a required field? Surprised we need a check here or the name fallback.

Reply:
updated 👍🏻